client/src/components/SearchBox/SearchBox.component.jsx

```diff
@@ -136,18 +136,23 @@ const SearchBox = ({
             >
               <InputGroup className="search-box">
                 {showSearchIcon ? (
-                  <InputLeftElement
-                    children={<BiSearch size="24" color="secondary.500" />}
-                    h="100%"
-                    w="24px"
-                    ml="19px"
-                    mr="20px"
+                  <InputRightElement
+                    bg="primary.500"
+                    children={<BiSearch size="24" color="white" />}
+                    h="40px"
+                    w="40px"
+                    mb="8px"
+                    mt="8px"
+                    mr="8px"
+                    borderRadius="4px"
+                    cursor="pointer"
+                    onClick={() => handleSubmit(inputValue)}
                   />
                 ) : null}
                 <Input
                   variant={showSearchIcon ? 'unstyled' : undefined}
```

Comment:
```suggestion
                  variant="unstyled"
```

to get rid of the white box outlined under Question Title input box

![image](https://user-images.githubusercontent.com/56983748/139397379-189b3bfa-d917-4f59-9fbd-375ce43e4ca5.png)